src/components/dar_collection_table/DarCollectionTableCellData.js

```diff
@@ -133,15 +133,20 @@ export function statusCellData({status = '- -', darCollectionId, label = 'status
   };
 }
 
-export function consoleActionsCellData({collection, reviewCollection, goToVote, showConfirmationModal, consoleType, relevantDatasets, resumeCollection}) {
+export function consoleActionsCellData({collection, reviewCollection, goToVote, showConfirmationModal, consoleType, relevantDatasets, resumeCollection, actions}) {
   let actionComponent;
 
   switch (consoleType) {
     case consoleTypes.ADMIN:
       actionComponent = h(AdminActions, {collection, showConfirmationModal});
       break;
     case consoleTypes.CHAIR:
-      actionComponent = h(ChairActions, {collection, showConfirmationModal, goToVote, relevantDatasets});
+      actionComponent = h(ChairActions, {
+        collection, showConfirmationModal, goToVote, relevantDatasets,
+        openEnabled: actions.includes('Open'),
+        cancelEnabled: actions.includes('Cancel'),
+        voteEnabled: actions.includes('Vote')
+      });
```

Comment:
I personally don't have a strong opinion, but since we're being told what actions are acceptable from the backend, would it make sense to consolidate `AdminActions`, `ChairActions`, etc., into one `Actions` class which displays the actions the backend says are available? That way the frontend is more separate from the backend.

Reply:
I considered doing that initially, but the ChairActions object was really complex, with further logic that controls what the state of the button should be (based on failure of request). I wasn't sure if the other Action components would also have a lot of specific logic to their state, but assumed they had to since they were split out in this way in the first place 🤔